pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1007,6 +1007,9 @@ public CompletableFuture<Optional<Topic>> getTopic(final TopicName topicName, bo
             CompletableFuture<Optional<Topic>> topicFuture = topics.get(topicName.toString());
             if (topicFuture != null) {
                 if (topicFuture.isCompletedExceptionally()
+                        && FutureUtil.getException(topicFuture).get().equals(FAILED_TO_LOAD_TOPIC_TIMEOUT_EXCEPTION)) {
+                    return FutureUtil.failedFuture(FAILED_TO_LOAD_TOPIC_TIMEOUT_EXCEPTION);
```

Comment:
If we don't remove the topic from topics, it seems it will lead to always unable to load the topic if first time load topic timeout?

Reply:
No, when the topic load timeout, it will be removed here.

https://github.com/apache/pulsar/pull/22739/files#diff-0210356c8a88e4efa89eb769a027fa6c166db479dbad8bbbbc704c6ed6e317f5L1746-R1751